<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WkfUnitsWidget</class>
 <widget class="QWidget" name="WkfUnitsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Units</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout" columnstretch="5,7">
     <item row="0" column="1">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Default Format</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Latitude / Longitude</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="latLonComboBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Time</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="timeComboBox"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="dateComboBox"/>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_2" columnstretch="5,5,1">
     <item row="14" column="2">
      <widget class="QSpinBox" name="powerDBSpinBox">
       <property name="maximum">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item row="16" column="0">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Ratio</string>
       </property>
      </widget>
     </item>
     <item row="11" column="2">
      <widget class="QSpinBox" name="massSpinBox">
       <property name="maximum">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="accelerationComboBox"/>
     </item>
     <item row="17" column="1">
      <widget class="QComboBox" name="speedComboBox"/>
     </item>
     <item row="18" column="0">
      <widget class="QLabel" name="labelTorque">
       <property name="text">
        <string>Torque</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="angleComboBox"/>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="angularRateComboBox"/>
     </item>
     <item row="17" column="2">
      <widget class="QSpinBox" name="speedSpinBox">
       <property name="maximum">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QSpinBox" name="angularAccelerationSpinBox">
       <property name="maximum">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>Power</string>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QSpinBox" name="forceSpinBox">
       <property name="maximum">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QComboBox" name="lengthComboBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Acceleration</string>
       </property>
      </widget>
     </item>
     <item row="12" column="2">
      <widget class="QSpinBox" name="massTransferSpinBox">
       <property name="maximum">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Angular Acceleration</string>
       </property>
      </widget>
     </item>
     <item row="18" column="1">
      <widget class="QComboBox" name="torqueComboBox"/>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Pressure</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="angularAccelerationComboBox"/>
     </item>
     <item row="9" column="2">
      <widget class="QSpinBox" name="frequencySpinBox">
       <property name="maximum">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Default Unit</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QSpinBox" name="angularRateSpinBox">
       <property name="maximum">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="labelMassTransfer">
       <property name="text">
        <string>Mass Transfer</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Altitude (low)</string>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <widget class="QComboBox" name="pressureComboBox"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Value Type</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QComboBox" name="massComboBox"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Angular Rate</string>
       </property>
      </widget>
     </item>
     <item row="10" column="2">
      <widget class="QSpinBox" name="lengthSpinBox">
       <property name="maximum">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QComboBox" name="frequencyComboBox"/>
     </item>
     <item row="14" column="1">
      <widget class="QComboBox" name="powerDBComboBox"/>
     </item>
     <item row="4" column="2">
      <widget class="QSpinBox" name="angleSpinBox">
       <property name="maximum">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item row="17" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Speed</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="accelerationSpinBox">
       <property name="maximum">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QComboBox" name="powerComboBox"/>
     </item>
     <item row="16" column="1">
      <widget class="QComboBox" name="ratioComboBox"/>
     </item>
     <item row="18" column="2">
      <widget class="QSpinBox" name="torqueSpinBox">
       <property name="maximum">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_21">
       <property name="text">
        <string>Precision</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>Frequency</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Angle</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="lowAltitudeComboBox"/>
     </item>
     <item row="3" column="2">
      <widget class="QSpinBox" name="highAltitudeSpinBox">
       <property name="maximum">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QSpinBox" name="lowAltitudeSpinBox">
       <property name="maximum">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Mass</string>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>Power DB</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QComboBox" name="massTransferComboBox"/>
     </item>
     <item row="13" column="2">
      <widget class="QSpinBox" name="powerSpinBox">
       <property name="maximum">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QComboBox" name="forceComboBox"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Altitude (space)</string>
       </property>
      </widget>
     </item>
     <item row="16" column="2">
      <widget class="QSpinBox" name="ratioSpinBox">
       <property name="maximum">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Force</string>
       </property>
      </widget>
     </item>
     <item row="15" column="2">
      <widget class="QSpinBox" name="pressureSpinBox">
       <property name="maximum">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="highAltitudeComboBox"/>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Length</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_24">
       <property name="text">
        <string>Duration</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QComboBox" name="durationComboBox"/>
     </item>
     <item row="7" column="2">
      <widget class="QSpinBox" name="durationSpinBox">
       <property name="maximum">
        <number>8</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_23">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Preferences on this page are shared among all AFSIM applications</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>accelerationComboBox</tabstop>
  <tabstop>accelerationSpinBox</tabstop>
  <tabstop>lowAltitudeComboBox</tabstop>
  <tabstop>lowAltitudeSpinBox</tabstop>
  <tabstop>highAltitudeComboBox</tabstop>
  <tabstop>highAltitudeSpinBox</tabstop>
  <tabstop>angleComboBox</tabstop>
  <tabstop>angleSpinBox</tabstop>
  <tabstop>angularRateComboBox</tabstop>
  <tabstop>angularRateSpinBox</tabstop>
  <tabstop>angularAccelerationComboBox</tabstop>
  <tabstop>angularAccelerationSpinBox</tabstop>
  <tabstop>durationComboBox</tabstop>
  <tabstop>durationSpinBox</tabstop>
  <tabstop>forceComboBox</tabstop>
  <tabstop>forceSpinBox</tabstop>
  <tabstop>frequencyComboBox</tabstop>
  <tabstop>frequencySpinBox</tabstop>
  <tabstop>lengthComboBox</tabstop>
  <tabstop>lengthSpinBox</tabstop>
  <tabstop>massComboBox</tabstop>
  <tabstop>massSpinBox</tabstop>
  <tabstop>massTransferComboBox</tabstop>
  <tabstop>massTransferSpinBox</tabstop>
  <tabstop>powerComboBox</tabstop>
  <tabstop>powerSpinBox</tabstop>
  <tabstop>powerDBComboBox</tabstop>
  <tabstop>powerDBSpinBox</tabstop>
  <tabstop>pressureComboBox</tabstop>
  <tabstop>pressureSpinBox</tabstop>
  <tabstop>ratioComboBox</tabstop>
  <tabstop>ratioSpinBox</tabstop>
  <tabstop>speedComboBox</tabstop>
  <tabstop>speedSpinBox</tabstop>
  <tabstop>torqueComboBox</tabstop>
  <tabstop>torqueSpinBox</tabstop>
  <tabstop>latLonComboBox</tabstop>
  <tabstop>timeComboBox</tabstop>
  <tabstop>dateComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
